src/lib/y2users/linux/helpers.rb

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2issues"
+
+module Y2Users
+  module Linux
+    # Helper methods for readers and writers
+    module Helpers
```

Comment:
To be honest I do not like much any "Helpers" as it tends to grow a lot over time. Why not having simple mixing like "WithIssues" that adds the method for issues handling? I think it can be even useful generally in yast2 for other modules.

Reply:
Good point.